research/index.html

```diff
@@ -18,57 +18,59 @@
+          <p><img src="https://cdn.vox-cdn.com/thumbor/W4PO8Jygk63Xc6JIyjKqinSHqWA=/0x0:640x362/920x613/filters:focal(269x130:371x232):format(webp)/cdn.vox-cdn.com/uploads/chorus_image/image/58195665/Screen_Shot_2011-12-16_at_11.08.27_AM.0.jpg"></p>
+        </div>
       </div>
 
-      <div class="jumbotron">
-        <h1 class="display-6">Microsoft issues emergency Windows update for processor security bugs</h1>
-        <p class="lead">Microsoft is issuing a rare out-of-band security update to supported versions of Windows today. The software update is part of a number of fixes that will protect against a newly-discovered processor bug in Intel, AMD, and ARM chipsets. </p>
-        <p><img src="https://cdn.vox-cdn.com/thumbor/keNcXCZlEEJifDM__KJH-Ms5MfU=/0x0:765x524/920x613/filters:focal(322x201:444x323):format(webp)/cdn.vox-cdn.com/uploads/chorus_image/image/58195111/windowsbluestock.0.jpg"></p>
-      </div>
+      <div class="card article-card">
+          <div class="card-body">
+            <h1 class="card-title">Microsoft issues emergency Windows update for processor security bugs</h1>
+            <p class="card-subtitle mb-2 text-muted">Microsoft is issuing a rare out-of-band security update to supported versions of Windows today. The software update is part of a number of fixes that will protect against a newly-discovered processor bug in Intel, AMD, and ARM chipsets.</p>
+            <p><img src="https://cdn.vox-cdn.com/thumbor/keNcXCZlEEJifDM__KJH-Ms5MfU=/0x0:765x524/920x613/filters:focal(322x201:444x323):format(webp)/cdn.vox-cdn.com/uploads/chorus_image/image/58195111/windowsbluestock.0.jpg"></p>
+          </div>
+        </div>
+
+      
 
-      <div class="jumbotron">
-        <h1 class="display-6">LG launches pink V30 ahead of CES</h1>
-        <p class="lead">LG’s pre-CES announcements have so far been high on technology and low on frills, but today the company goes the other way by issuing a new color for its flagship V30 smartphone.</p>
-        <p><img src="https://cdn.vox-cdn.com/thumbor/kmPa0HRLLPOr_xCM0Ep8_kNef_o=/0x0:1800x1200/920x613/filters:focal(756x456:1044x744):format(webp)/cdn.vox-cdn.com/uploads/chorus_image/image/58188111/v30_2.0.jpg"></p>
+      <div class="card article-card">
+        <div class="card-body">
+          <h1 class="card-title">LG launches pink V30 ahead of CES</h1>
+          <p class="card-subtitle mb-2 text-muted">LG’s pre-CES announcements have so far been high on technology and low on frills, but today the company goes the other way by issuing a new color for its flagship V30 smartphone.</p>
+          <p><img src="https://cdn.vox-cdn.com/thumbor/kmPa0HRLLPOr_xCM0Ep8_kNef_o=/0x0:1800x1200/920x613/filters:focal(756x456:1044x744):format(webp)/cdn.vox-cdn.com/uploads/chorus_image/image/58188111/v30_2.0.jpg"></p>
+        </div>
       </div>
      </section>
 
-     
 
-    <footer class="footer">
+    <footer class="footer footer-content-wrapper">
```

Comment:
Nesting `<div>` for dividing classes is very effective for styling. Check this snippet:
```
<footer class="footer">
    <div class="footer-content-wrapper">
        <!-- Some tags here -->
    </div>
</footer>
```

Reply:
Thank you so much!!!!!!!!  😃